packages/protocol/contracts/governance/interfaces/IValidators.sol

```diff
@@ -5,4 +5,6 @@ interface IValidators {
   function isVoting(address) external view returns (bool);
   function isValidating(address) external view returns (bool);
   function getValidators() external view returns (address[] memory);
+  function validatorAddressFromCurrentSet(uint256) external view returns (address);
```

Comment:
Do we need this in the IValidators interface?

Reply:
Leftover from before moving the precompile into Attestations.

Reply:
So remove it?